pyanaconda/modules/boss/installation.py

```diff
@@ -145,7 +145,9 @@ def _copy_kickstart(self):
                 "/run/install/ks.cfg",
                 "/root/original-ks.cfg"
             )
-            os.chmod(join_paths(self._sysroot, "/root/original-ks.cfg"), 0x0600)
+            result_ks_path = join_paths(self._sysroot, "/root/original-ks.cfg")
+            if os.path.exists(result_ks_path):
```

Comment:
This is confusing. If the original kickstart file doesn't exist, we shouldn't copy it in the first place.

Reply:
The copying function checks for that itself. It's just that here we have to do more. Maybe it's a better idea to inline it all?